lib/mixpanel-node.js

```diff
@@ -41,7 +41,7 @@ var create_client = function(token, config) {
         var event_data = new Buffer(JSON.stringify(data));
         var request_data = {
             'data': event_data.toString('base64'),
-            'ip': 0,
+            'ip': metrics.config.track_ip ? 1 : 0,
```

Comment:
As this is a non-standard use-case, it might add clarity to name the config something like 'metrics.config.track_server_ip'